docs/development/tooling.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Tooling'
+date: 2022-01-28T00:00:00+00:00
+weight: 40
+geekdocRepo: https://github.com/owncloud/web
+geekdocEditPath: edit/master/docs/development
+geekdocFilePath: tooling.md
+---
+
+{{< toc >}}
+
+## Packaging
+
+Web is using [pnpm](https://pnpm.io/) as package manager and [vite](https://vitejs.dev/) as build tool. The latter is built on top of [rollup](https://rollupjs.org/) and brings some additional features such as instant hot-reloading.
+
+## Development Setup
+
+### Prerequisites
+
+- docker
+- docker-compose (if not already included in your docker installation)
+- pnpm
+- node
+
+If you’re not using Docker Desktop, you might have to modify your `/etc/hosts` and add `127.0.0.1 host.docker.internal` to make `host.docker.internal` links work.
+
+### Installing dependencies
+
+After cloning the source code, install the dependencies via `pnpm install`.
+
+### Starting the server
+
+Web supports oCIS and oC10 as servers. You can start both of them by running `docker-compose up ocis oc10`.
+
+Note that `ocis` needs a short while to start because it is waiting for `tika` to be initialized. This is the case as soon as the `tika-service` container has stopped running.
+
+### Building and accessing Web
+
+After the docker containers are running (and `tika` is being initialized), run `pnpm build:w` to build Web. This also includes hot-reloading after changes you make, although it will take a while to rebuild the project. See down below for some details on how to enable instant hot-reloading.
+
+Now you can access Web via https://host.docker.internal:9200 (oCIS) and http://host.docker.internal:8080 (OC10).
+
+### Using instant hot-reload via vite
```

Comment:
```suggestion
### Installing Dependencies

After cloning the source code, install the dependencies via `pnpm install`.

### Starting the Server

Web supports oCIS and oC10 as servers. You can start both of them by running `docker-compose up ocis oc10`.

Note that `ocis` needs a short while to start because it is waiting for `tika` to be initialized. This is the case as soon as the `tika-service` container has stopped running.

### Building and Accessing Web

After the docker containers are running (and `tika` is being initialized), run `pnpm build:w` to build Web. This also includes hot-reloading after changes you make, although it will take a while to rebuild the project. See down below for some details on how to enable instant hot-reloading.

Now you can access Web via https://host.docker.internal:9200 (oCIS) and http://host.docker.internal:8080 (OC10).

### Using Instant Hot-Reload via Vite
```
Just some headline capitalisation stuff